lit-base/model_service.py

```diff
@@ -0,0 +1,91 @@
+import mxnet as mx
+import tempfile
+import urllib2
+import numpy as np
+from PIL import Image
+
+from model_loader import sym, arg_params, aux_params
+
+from collections import namedtuple
+from StringIO import StringIO
+
+
+ctx = mx.cpu()
+
+
+def preprocess(event):
+    """
+    Preprocess Lambda event into input data format for mx mod.
+    :return: input data NDArray
+    event : dict
+        Lambda event
+    """
+    img = None
+    try:
+        # GET method
+        if event.get('httpMethod') == 'GET':
+            url = event['queryStringParameters']['url']
+            # download image from url
+            img_file = tempfile.NamedTemporaryFile(delete=True)
+            if url:
```

Comment:
its fine for a demo(not asking to remove right now) 

should you support url? 
might create an opportunity for someone to just pass a URL and DDOS this service. All I need is a small lambda function that creates 100s of requests point at some large binary. You can bring down two services the one which hosts the binary and this lambda.
@kalyc @frankfliu what do you think?

Reply:
you should leave it to the client code to fetch from the host/service and it pass it you.

Reply:
@nswamy Agree! Modified accordingly in lit-mar